TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/EDGE_DETECTION.py

```diff
@@ -0,0 +1,66 @@
+from flojoy import flojoy, DataContainer
+import cv2
+import numpy as np
+from PIL import ImageFilter, Image
+
+
+@flojoy
+def EDGE_DETECTION(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """
+    Apply edge detection operation on the input `DataContainer` class,
```

Comment:
Can you please follow the guideline mentioned here for docstring: https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md?plain=1#L19